source/sass/print.scss

```diff
@@ -0,0 +1,58 @@
+@media print {
+  @page {
+    margin: 2cm;
+  }
+
+  body {
+    color: $black;
+    background: $white;
+    width: 100%;
+  }
+
+  video {
```

Comment:
we probably want to add `iframe` here, too.

Reply:
hide video and iframe on print mode (will update PR to include `iframe`)